src/pages/api/database/cassandra.ts

```diff
@@ -0,0 +1,35 @@
+import { ColumnSchema } from "../models/column";
+import { TaskSchema } from "../models/task";
+
+const ExpressCassandra = require("express-cassandra");
+
+const cassandraClient = ExpressCassandra.createClient({
+  clientOptions: {
+    contactPoints: ["localhost"],
+    protocolOptions: { port: 9042 },
+    keyspace: "mi_keyspace",
+    localDataCenter: "datacenter1",
+    queryOptions: { consistency: ExpressCassandra.consistencies.one },
+  },
+  ormOptions: {
+    defaultReplicationStrategy: {
+      class: "SimpleStrategy",
+      replication_factor: 1,
+    },
+    migration: "safe",
+    createKeyspace: true,
+    dropTableOnSchemaChange: false,
+  },
+});
+
+
+export const ColumnModel = cassandraClient.loadSchema("columns", ColumnSchema);
+export const TaskModel = cassandraClient.loadSchema("tasks", TaskSchema);
+
+[ColumnModel, TaskModel].forEach((model: any) => {
```

Comment:
Move this file away from `api` folder. Everything inside that can be called by a client.